### PR TITLE
Kl 2023 02 moderation rules

### PR DESCRIPTION
**Pull request by fuzzylogic2000**

### CODE REVIEW

File: meinberlin/apps/budgeting/rules.py [hunk 1]
```diff
@@ -45,7 +46,10 @@
 
 rules.add_perm(
     "meinberlin_budgeting.moderate_proposal",
-    module_predicates.is_context_moderator | module_predicates.is_context_initiator,
+    is_superuser
+    | module_predicates.is_context_moderator
+    | module_predicates.is_context_initiator
+    | module_predicates.is_context_group_member,
```
Comment (Rineee) [inline meinberlin/apps/budgeting/rules.py:52]: Couldnt we use ```is_allowed_moderate_project``` here? https://github.com/liqd/adhocracy4/blob/ca0331aa7612304691fe637903fec4f6936a1f9f/adhocracy4/modules/predicates.py#L87

**Comment by Rineee** [comment]
Yes, cool, just one question!

File: meinberlin/apps/budgeting/rules.py (hunk 1 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/budgeting/rules.py:52]: Yes! It does exactly do that. Will do.

File: meinberlin/apps/budgeting/rules.py [hunk 2]
```diff
@@ -45,7 +46,7 @@
 
 rules.add_perm(
     "meinberlin_budgeting.moderate_proposal",
-    module_predicates.is_context_moderator | module_predicates.is_context_initiator,
+    is_superuser | module_predicates.is_allowed_moderate_project,
```
Comment (Rineee) [inline meinberlin/apps/budgeting/rules.py:49]: But the ```is_allowed_moderate_project``` already includes the superusers, right?

**Comment by Rineee** [comment]
@fuzzylogic2000 Another question, soorry..

File: meinberlin/apps/budgeting/rules.py (hunk 2 above)
Comment (fuzzylogic2000) [reply meinberlin/apps/budgeting/rules.py:49]: How did I not see that?

**Pull request merged**